build/steps-selfcontained.yml

```diff
@@ -0,0 +1,45 @@
+##
+# Pack IQ# self-contained executable.
+##
+
+steps:
+
+##
+# Pre-reqs
+##
+- task: DownloadBuildArtifacts@0
+  inputs:
+    artifactName: iqsharp
+    downloadPath: '$(System.DefaultWorkingDirectory)/artifact'
+
+- pwsh: |
+    Move-Item "$Env:SYSTEM_DEFAULTWORKINGDIRECTORY/artifact/iqsharp/drops" "$Env:SYSTEM_DEFAULTWORKINGDIRECTORY/drops"
+  displayName: "Copy build artifacts to IQ# working directory"
+
+- task: CondaEnvironment@1
+  inputs:
+    packageSpecs: "python=3.7 pip setuptools pytest jupyter"
+  displayName: 'Use conda environment w/ Python 3.'
```

Comment:
Is this used by the self-contained job?

Reply:
I don't think so. I'll remove it and verify. Thanks for noticing this, will save a little bit of time in the build!